components/automate-chef-io/content/docs/quickstart.md

```diff
@@ -14,46 +14,71 @@ toc = true
 Hello and welcome to Chef Automate 2! This document guides you through the initial installation and trial activation.
 
 The easiest way to start exploring Chef Automate is to install it on a local VM.
-We recommend using [Vagrant][] to create your test environment.
+We recommend using [Vagrant](https://www.vagrantup.com/) to create your test environment.
 Use this Vagrantfile to provision a VM. Chef recommends a minimum of 4GB RAM.
 
+## Add an entry to /etc/hosts
+
+The example Vagrantfile sets the VM hostname to `chef-automate.test`. The installer uses this hostname as the FQDN that Chef Automate listens on.
+For security reasons, you must access Chef Automate's web UI using the configured FQDN.
+To avoid configuring DNS for your test instance, we recommend adding an entry to your workstation's `/etc/hosts` file.
+Use the following command to append the required entry:
+
+```bash
+echo 192.168.33.199 chef-automate.test | sudo tee -a /etc/hosts
+```
+
 ## Create a Vagrantfile
 
 Copy and paste the command below to create a `Vagrantfile`:
 
 ```ruby
-cat > Vagrantfile <<EOH
-# Put the contents of this snippet in a file named Vagrantfile
+cat > Vagrantfile <<'EOH'
+CFG_BOX      = 'bento/ubuntu-18.04'
+CFG_IP       = '192.168.33.199'
+CFG_HOSTNAME = 'chef-automate.test'
+
+$deployscript = <<-SCRIPT
+export DEBIAN_FRONTEND='noninteractive'
+apt-get -qq update
+apt-get install -y --no-install-recommends apt-transport-https ca-certificates curl gnupg-agent software-properties-common unzip
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+apt-get update
+apt-get install -y --no-install-recommends docker-ce docker-ce-cli containerd.io
+apt-get clean
+groupadd docker
+usermod -aG docker vagrant
+sysctl -w vm.max_map_count=262144
+sysctl -w vm.dirty_expire_centisecs=20000
+echo "${CFG_IP} ${CFG_HOSTNAME}" | tee -a /etc/hosts
+curl -fsSL https://packages.chef.io/files/current/automate/latest/chef-automate_linux_amd64.zip -o /tmp/chef-automate_linux_amd64.zip
+unzip -qod /usr/local/bin /tmp/chef-automate_linux_amd64.zip
+chmod +x /usr/local/bin/chef-automate
+chef-automate deploy --accept-terms-and-mlsa
```

Comment:
Going to leave this as a question for @sdelano 
Are we allowed to accept the terms and mlsa like this

Reply:
We should not do that, no

Reply:
@avoidik can you remove that too then please?

Reply:
@susanev i think that pretty much puts us back to what it was before. I think the intent was that the user just types `vagrant up` and they have a working automate

Reply:
@sdelano not sure if this is ok, but we might be able to request people read and accept the mlsa right before vagrant provisions. It'd be something along the lines of asking people to read https://www.chef.io/online-master-agreement/ and type accept. We'd still then provide `--accept-terms-and-mlsa` to the command line in the script, but we would have got a positive confirmation.

Reply:
@jaym we could also do something along the lines of `ACCEPT_CHEF_TERMS_AND_MLSA=true vagrant up`

Reply:
Obviously we'd then have to look for that environment variable in the vagrant command, but it would allow the user to type their intent in but also not have to pause the deployment for user input.

Alternatively, the vagrant file is just Ruby code, so it could prompt the user for input before doing all the work. I'm not sure which one I like better.

Reply:
@susanev thoughts on the above options?

Reply:
i think both. If we don't see `ACCEPT_CHEF_TERMS_AND_MLSA=true`, then the current script will fail when it tries to request user input.